### ISSUE TITLE
About deraining

### ISSUE THREAD

**Issue by kiriiiia** [swz30/MPRNet#125]
In the paper，you say we compute image quality scores using the Y channel (in YCbCr color space), but in training or testing code, i didn't see it computed on YCBCR color space. Did i missing the code？

**Comment by Feecuin** [comment]
Hi, can you take a look at the MPRNet training code, my training loss doesn't go down every time.
